src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomInt = random.nextInt(100);
+        return new Ball(ColorSupplier.getRandomColor(), randomInt);
```

Comment:
The 'ColorSupplier.getRandomColor()' method is being called statically which is against the checklist. You should create an instance of 'ColorSupplier' and call the 'getRandomColor' method on that instance.